external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -636,4 +675,45 @@ public String toString() {
     private String getTopicsString() {
         return kafkaSpoutConfig.getSubscription().getTopicsString();
     }
+
+    public static class Info {
```

Comment:
Nit: Put this in the internal package since it's used by multiple classes. Maybe call it CommitMetadata or something like that.

Reply:
Done